File: seongwon/Programmers/Level_0/Lv0_배열_만들기_3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> intervals) {
    vector<int> answer;
    
    for(int i = intervals[0][0]; i <= intervals[0][1]; i++){
        answer.push_back(arr[i]);    
    }
    
    for(int i = intervals[1][0]; i <= intervals[1][1]; i++){
        answer.push_back(arr[i]);    
    }
    return answer;
}

File: dohyeon/Programmers/Level_0/Lv0_배열_만들기_3.py
def solution(arr, intervals):
    answer = []
    for i in intervals :
        for idx, value in enumerate(arr) :
            if idx >= i[0] and idx <= i[1] :
                answer.append(value)
    return answer

print(solution([1, 2, 3, 4, 5], [[1, 3], [0, 4]]))